tests/test_selection/test_drop_high_psi_features.py

```diff
@@ -0,0 +1,543 @@
+        n_redundant=2,
+        n_clusters_per_class=1,
+        weights=[0.50],
+        class_sep=2,
+        random_state=1,
+    )
+
+    # transform array into pandas df
+    colnames = ["var_" + str(i) for i in range(6)]
+    X = pd.DataFrame(X, columns=colnames)
+
+    # Add drifted features that will be dropped during transformation.
+    X["drift_1"] = [number for number in range(X.shape[0])]
+    X["drift_2"] = [number / 2 for number in range(X.shape[0])]
+
+    return X
+
+
+@pytest.fixture(scope="module")
+def df_mixed_types():
+    df = pd.DataFrame(
+        {
+            "A": [it for it in range(0, 20)],
+            "B": [1, 2, 2, 1] * 5,
+            "C": ["A", "B", "D", "D"] * 5,
+            "time": [date(2019, 1, it + 1) for it in range(20)],
+        }
+    )
+
+    return df
+
+
+# ====  test  main functionality of the class ====
+def test_fit_attributes(df):
+    """Check the value of the fit attributes.
+
+    The expected PSI values used in the assertion were determined using
+    the Probatus package.
+    ```
+    from probatus.stat_tests import AutoDist
+    psi_calculator = AutoDist(statistical_tests=["PSI"],
+                    binning_strategies="QuantileBucketer",
+                    bin_count=10)
+    train_df = data.iloc[0:500,:]
+    test_df = data.iloc[500:, :]
+    psi = psi_calculator.compute(train_df, test_df)
+    ```
+    """
+    transformer = DropHighPSIFeatures()
+    transformer.fit_transform(df)
+
+    expected_psi = {
+        "var_0": 0.043828484052281,
+        "var_1": 0.040929870747665395,
+        "var_2": 0.04330418495156895,
+        "var_3": 0.03773286532548153,
+        "var_4": 0.05047388515663041,
+        "var_5": 0.014717735595712466,
+        "drift_1": 8.283089355027482,
+        "drift_2": 8.283089355027482,
+    }
+
+    assert transformer.variables_ == [
+        "var_0",
+        "var_1",
+        "var_2",
+        "var_3",
+        "var_4",
+        "var_5",
+        "drift_1",
+        "drift_2",
+    ]
+    assert transformer.psi_values_ == pytest.approx(expected_psi, 12)
+    assert transformer.features_to_drop_ == ["drift_1", "drift_2"]
+    assert transformer.n_features_in_ == 8
+
+
+# ================ test init parameters =================
+
+# Define two dictionaries with arguments: one with default values and
+# one with arbitrary values.
+default_dict = {
+    "split_col": None,
+    "split_frac": 0.5,
+    "split_distinct": False,
+    "cut_off": None,
+    "switch": False,
+    "threshold": 0.25,
+    "bins": 10,
+    "strategy": "equal_frequency",
+    "min_pct_empty_bins": 0.0001,
+    "missing_values": "raise",
+    "variables": None,
+}
+
+args_dict = {
+    "split_col": None,
+    "split_frac": 0.5,
+    "split_distinct": False,
+    "cut_off": None,
+    "switch": False,
+    "threshold": 0.25,
+    "bins": 10,
+    "strategy": "equal_frequency",
+    "min_pct_empty_bins": 0.0001,
+    "missing_values": "raise",
+    "variables": None,
+}
```

Comment:
the 2 dictionaries are identical... this is probably a bug? we want to test that different parameter values are correctly assigned.